src/bundle/Resources/public/js/scripts/core/draggable.js

```diff
@@ -9,22 +9,11 @@
         attachCustomEventHandlersToItem: () => {},
```

Comment:
Now when we can override methods in this class those "after" callback are not needed.